examples/LogLevelsLogger/LogLevelsLogger.ino

```diff
@@ -37,84 +37,72 @@ void setup()
   // set serial port appender
   rf24Logger.setHandler(&rf24SerialLogHandler);
 
-  rf24Logger.info((const __FlashStringHelper*) vendorID, F("RF24Log/examples/LogLevelsLogger"));
+  RF24LOGGER_info(vendorID, "RF24Log/examples/LogLevelsLogger");
 }
 
 void loop()
 {
   Serial.println("Set log level to OFF");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::OFF);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to ERROR");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::ERROR);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to ERROR + 1");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::ERROR + 1);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to ERROR + 7");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::ERROR + 7);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to WARN");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::WARN);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to INFO");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::INFO);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to DEBUG");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::DEBUG);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to DEBUG + 1");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::DEBUG + 1);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to DEBUG + 7");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::DEBUG + 7);
-  logSimpleRamMessage();
+  logSimpleMessage();
 
   Serial.println("Set log level to ALL");
   rf24SerialLogHandler.setLogLevel(RF24LogLevel::ALL);
-  logSimpleRamMessage();
+  logSimpleMessage();
```

Comment:
I love this example, but it could be simplified (and much more interactive) if we allow the user to change the `logLevel` using the Serial terminal. That we could dynamically demonstrate filtering.

Reply:
Yes, with new changes it is a bit irrelevant where the message will be stored (RAM or FLASH)  - from the point of the unified logging library - as it will be decided at compile time depending on the architecture (AVR or non-AVR).

Interactive examples is a cool idea. I have created #12 for this.